worker/firewaller/firewaller.go

```diff
@@ -976,7 +976,7 @@ func (fw *Firewaller) forgetMachine(machined *machineData) error {
 	// Unusually, it's fine to ignore this error, because we know the machined
 	// is being tracked in fw.catacomb. But we do still want to wait until the
 	// watch loop has stopped before we nuke the last data and return.
-	worker.Stop(machined)
+	_ = worker.Stop(machined)
```

Comment:
`_ = worker.Stop(machined)` out of curiosity, why the `_ =`?

Reply:
John brought this up on one of my recent PRs. I have started doing this if unchecked errors are near code that I am touching.

Here, there is an comment precisely about the fact that they are not checked; but in all cases I figure it is better to indicate that they are being ignore in code semantics, than hope that everyone has a linter that highlights them. That way they might prompt consideration if someone is debugging a code path that they're on.

Reply:
Nice!